ConfigSpace/hyperparameters_/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .hyperparameter import Hyperparameter
+
+__all__ = [Hyperparameter]
```

Comment:
Unfortunatly there have to be strings, Copilot has been great for doing this for me heh heh